plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.test.tsx

```diff
@@ -169,10 +169,6 @@ describe('DefaultCatalogPage', () => {
 
     expect(columnHeaderLabels).toEqual([
       'Name',
-      'System',
-      'Owner',
-      'Type',
-      'Lifecycle',
```

Comment:
This is the default case right? It would be nice to have tests for the other types.